### DISCUSSION TITLE
Ability to Customize UI language and Apple Music locale

### DISCUSSION THREAD

**Discussion by s-kk** [ciderapp/Cider#1306] [Feature Request]
Dear Cider developers, 
Please consider adding the feature to customize UI language and Apple Music locale separately.  

For example, I might be using an OS with English as my UI language. However, with Cider being English, lots of Japanese song titles are being converted to Romaji (the romanization of the Japanese written language), which is annoying to read. If I switch Cider language to Japanese, there would mixed translations of both English and Japanese (since the i18n progress is not up-to-date with the development), which makes the UI harder to use.  

Additionally, I would prefer keeping the Cider language consistent with my OS language, without sacrificing the readability of Japanese song titles (this issue could also occur for songs titles in other languages).  

Any help is appreciated!

**Comment by maikirakiwi** [comment]
I'm working on this exact feature to show Song Titles and Albums in their original language.

Will likely appear in Cider 2 though.
